tests/data_model/wikibase/test_value_types.py

```diff
@@ -42,6 +42,13 @@ class TestDateTimeValue:
     def test_equivalence(self, statement, equivalent):
         assert DateTimeValue(statement) == equivalent
 
+    @pytest.mark.parametrize("statement,equivalent", [
+        (mock["statement"]["with_datetime"]["invalid_date_day_precision"], "+1986-05-04T00:00:00Z"),
```

Comment:
They would be false anyway as they are not the same dates, I think the test should be more explicit in checking that the eq function doesn't throw exceptions for invalid dates, otherwise it's not necessarily clear what you are testing here.

Reply:
Can you expand on how to do this a bit? This test will always be pass if the "upstream date" is valid so it's not possible to have the dates both be equal and test the Wikibase bad date will not fail.

I added a 3rd test case to check that it also doesn't throw when "the same" invalid date is in both places. Also renamed to makes test more explicit.

I'm not sure how to test "it doesn't throw" since that is really a prerequisite of any passing pytest case

Reply:
I think this is great, thank you! :)